cfgov/v1/models/molecules.py

```diff
@@ -12,121 +12,90 @@ def isRequired(field_name):
-            error_dict.update({'image': img_err, 'image_path': img_err, 'image_alt': isRequired('Image alt')})
-
-        if block_data['image'] and block_data['image_path']:
-            img_err = ['Please select one method of image rendering']
-            error_dict.update({
-                'image': img_err,
-                'image_path': img_err})
-
-        if block_data['image_path'] and not block_data['image_alt']:
-            error_dict.update({'image_alt': isRequired('Image Alt')})
-
-        if error_dict:
-            raise ValidationError("ImageText5050 validation errors", params=error_dict)
-        else:
-            return block_data
+    link = atoms.Hyperlink()
 
     class Meta:
         icon = 'image'
-        template = 'v1/demo/molecules/image_text_5050.html'
+        template = '_includes/molecules/image-text-50-50.html'
 
 
 class ImageText2575(blocks.StructBlock):
     heading = blocks.CharBlock(max_length=100, required=True)
     body = blocks.RichTextBlock(required=True)
-    image = ImageChooserBlock(required=False)
-    image_path = blocks.CharBlock(required=False)
-    image_alt = blocks.CharBlock(required=False)
-    link_url = blocks.URLBlock(required=False)
-    link_text = blocks.CharBlock(max_length=100, required=False)
+    image = atoms.ImageBasic()
+    link = atoms.Hyperlink()
     has_rule = blocks.BooleanBlock(required=False)
 
-    def clean(self, data):
-        error_dict = {}
-        try:
-            block_data = super(ImageText5050, self).clean(data)
-        except ValidationError as e:
-            error_dict.update(e.params)
-            block_data = data
-
-        if not block_data['image'] and not block_data['image_path'] and not block_data['image_alt']:
-            img_err = ['Please upload or enter an image path']
-            error_dict.update({'image': img_err, 'image_path': img_err, 'image_alt': isRequired('Image alt')})
-
-        if block_data['image'] and block_data['image_path']:
-            img_err = ['Please select one method of image rendering']
-            error_dict.update({
-                'image': img_err,
-                'image_path': img_err})
-
-        if block_data['image_path'] and not block_data['image_alt']:
-            error_dict.update({'image_alt': isRequired('Image Alt')})
-
-        if error_dict:
-            raise ValidationError("ImageText2575 validation errors", params=error_dict)
-        else:
-            return block_data
-
     class Meta:
         icon = 'image'
-        template = 'v1/wagtail/molecules/image_text_2575.html'
+        template = '_includes/molecules/image-text-25-75.html'
 
 
 class TextIntroduction(blocks.StructBlock):
     heading = blocks.CharBlock(max_length=100, required=True)
     intro = blocks.CharBlock(max_length=100, required=True)
     body = blocks.RichTextBlock(required=False)
-    link_url = blocks.URLBlock(required=False)
-    link_text = blocks.CharBlock(max_length=100, required=False)
+    link = atoms.Hyperlink()
```

Comment:
Can we make this link not-required?

![image](https://cloud.githubusercontent.com/assets/1860176/11284060/cb2514dc-8ed5-11e5-830e-979c42bf9983.png)